packages/rollup-full-node/src/app/test-web3-rpc-handler.ts

```diff
@@ -132,8 +134,24 @@ export class TestWeb3Handler extends DefaultWeb3Handler {
    *
    * @param The transaction to send
    */
-  public async sendTransaction(ovmTx: object): Promise<string> {
-    return this.sendRawTransaction(rlpEncodeTransaction(ovmTx))
+  public async sendTransaction(ovmTx: any): Promise<string> {
+    if (!ovmTx.nonce) {
+      ovmTx.nonce = await this.getTransactionCount(ovmTx.from, latestBlock)
+    }
+    if (!ovmTx.to) {
+      ovmTx.to = ZERO_ADDRESS
```

Comment:
It might be slightly cleaner to set this to `null` instead? Like I actually don't know what clients normally do when decoding / encoding Ethereum transactions, but the way contract creations normally work is we end up decoding the tx and get a `to: null` in the result, I believe. I want to know exactly how Ethereum encodes EOA contract creation though....

Though after thinking about it, this approach does actually work so could also keep it. Maybe it would be good to have a test which does a contract creation too? My only thought is if we ever move away from using the weird `to=ZERO_ADDRESS` to signify a contract creation this could fail silently.

Reply:
did this because the execution manager checks for `isCreate` with `target===ZERO_ADDRESS`, but will def add a test